renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -174,6 +176,115 @@ class EGLHook : LibraryHook, public GLPlatform
       eglDestroyContext_real(context.egl_dpy, context.egl_ctx);
   }
 
+  void DeleteReplayContext(GLWindowingData context)
+  {
+    if(eglDestroyContext_real)
+    {
+      eglMakeCurrent_real(context.egl_dpy, 0L, 0L, NULL);
+      eglDestroyContext_real(context.egl_dpy, context.egl_ctx);
+    }
+  }
+
+  void SwapBuffers(GLWindowingData context)
+  {
+    eglSwapBuffers_real(context.egl_dpy, context.egl_wnd);
+  }
+
+  void GetOutputWindowDimensions(GLWindowingData context, int32_t &w, int32_t &h)
+  {
+    eglQuerySurface_real(context.egl_dpy, context.egl_wnd, EGL_WIDTH, &w);
+    eglQuerySurface_real(context.egl_dpy, context.egl_wnd, EGL_HEIGHT, &h);
+  }
+
+  bool IsOutputWindowVisible(GLWindowingData context)
+  {
+    GLNOTIMP("Optimisation missing - output window always returning true");
+
+    return true;
+  }
+
+  GLWindowingData MakeOutputWindow(WindowingSystem system, void *data, bool depth, void *share_context)
+  {
+    GLWindowingData ret;
+    EGLNativeWindowType window = 0;
+
+    if(system == eWindowingSystem_Xlib)
+    {
+      XlibWindowData *xlib = (XlibWindowData *)data;
+      window = (EGLNativeWindowType)xlib->window;
+    }
+    else if(system == eWindowingSystem_Unknown)
+    {
+      // allow undefined so that internally we can create a window-less context
+      Display *dpy = XOpenDisplay(NULL);
```

Comment:
I know this code is just moved not new, but I hadn't noticed before that this Display isn't used. I guess this won't work on android, so it will need to be deleted anyway?